src/setup.rs

```diff
@@ -267,19 +260,21 @@ pub fn init() -> Config {
         password = password.replace("\r", "");
         password = password.replace("\n", "");
         let password_hash = hash(&password);
-        let password_path = get_password_path();
         fs::write(&password_path, password_hash.as_slice()).unwrap_or_else(|e| {
             eprintln!("Can't write password: {}", e);
             process::exit(1);
         });
+    }
 
-        let mut config_path = data_dir;
-        config_path.push("config.toml");
-        eprintln!(
-            "Almost ready. To get started, edit the config file at {} and restart.",
-            &config_path.to_str().unwrap(),
-        );
-        process::exit(0);
+    let config_path = get_config_path();
+    if !config_path.exists() {
+        println!("Generating config file at {}", config_path.display());
+        let config = Config::default();
+        config.write_file().unwrap_or_else(|e| {
+            eprintln!("Couldn't write the config file: {}", e);
```

Comment:
That's really stupid but I'd prefer "Can't" to "Couldn't" to match the other error messages.

Reply:
Sure, keeping the tense the same makes sense. I changed an eprintln here to a println since it wasn’t an error, do you want all of these as eprintlns?

Reply:
No, the change makes sense there, I can use stderr a bit too automatically by moment. Can I change the formulation myself and merge ?

Reply:
Yeah sure, you’re always welcome to make any changes to my PRs, especially as we’re in very different time zones.